### PR TITLE
fix(kbadge): update hover behavior [KHCP-5973]

### PR DESCRIPTION
**Pull request by portikM**
# Summary

https://konghq.atlassian.net/browse/KHCP-5973

## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

**Comment by adamdehaven** [comment]
Might make sense to add a CSS transition to the hover state as well, something like `.2s ease-in-out`

**Comment by portikM** [comment]
> Might make sense to add a CSS transition to the hover state as well, something like `.2s ease-in-out`

Will try tomorrow but have a feeling the two won't play nice together.

File: src/components/KBadge/KBadge.vue [hunk 1]
```diff
@@ -338,6 +342,27 @@ watch(badgeText, () => {
 @import '@/styles/functions';
 
 .k-badge {
+  // default appearance colors local variables
```
Comment (kaiarrowood) [inline src/components/KBadge/KBadge.vue:345]: ❤️

**Review by kaiarrowood — APPROVED** [review]
Looks great, thank you for cleaning up the styles with the CSS vars! 🚀

**Pull request merged**